habitat-baselines/habitat_baselines/rl/ppo/ppo_trainer.py

```diff
@@ -340,11 +340,19 @@ def _compute_actions_and_step_envs(self, buffer_index: int = 0):
             )
 
             profiling_wrapper.range_push("compute actions")
+
+            # Obtain lenghts
+            step_batch_lens = {
```

Comment:
@ASzot are these changes in ppo_trainer okay, or do we need to make sure that they also work with VERTrainer?

Reply:
I think VERTrainer has not been updated in a while, I imagine the plan is to run the exps with PPO?

Reply:
Yeah that's right. Let's not worry about VER.